conf/application.conf

```diff
@@ -93,3 +93,6 @@ tracking-consent-frontend {
   gtm.container = "transitional"
 }
 
+feature {
```

Comment:
@chandraAndem As it is added to the application.conf, we may not need this config in app-config-base. please check

Reply:
Advantage of having in both places is, in application conf is useful for local testing and in App config base is useful when we have to make changes to Feature switch we do not need to make any repository change and make a release instead make config change and redeploy.